graylog2-web-interface/src/components/inputs/CreateInputControl.tsx

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+
+import * as React from 'react';
+import { useState } from 'react';
+import styled from 'styled-components';
+import { PluginStore } from 'graylog-web-plugin/plugin';
+
+import { useStore } from 'stores/connect';
+import { InputsActions } from 'stores/inputs/InputsStore';
+import { InputTypesActions, InputTypesStore } from 'stores/inputs/InputTypesStore';
+import type { InputDescription } from 'stores/inputs/InputTypesStore';
+import { getPathnameWithoutId } from 'util/URLUtils';
+import { TELEMETRY_EVENT_TYPE } from 'logic/telemetry/Constants';
+import useSendTelemetry from 'logic/telemetry/useSendTelemetry';
+import useLocation from 'routing/useLocation';
+import { Col, Row, Button } from 'components/bootstrap';
+import { ExternalLinkButton, Select } from 'components/common';
+import { InputForm } from 'components/inputs';
+import type { Input } from 'components/messageloaders/Types';
+
+const NewInputRow = styled(Row)`
+  margin-bottom: 8px;
```

Comment:
Can we use theme spacing here?

Reply:
Actually I realised this is obsolete as this styling is overridden by the content class